tests/core/test_test.py

```diff
@@ -501,6 +632,39 @@ def test_partial_data_column_order(sushi_context: Context) -> None:
         ).run()
     )
 
+    _check_successful_or_raise(
```

Comment:
This doesn't seem particularly useful, CSVs have a fix column order right? Whereas in YAML you can mix it up as shown in the test above.